app/client/components/footer/footer.tsx

```diff
@@ -221,7 +213,7 @@ export const Footer = () => {
               <div css={emailSignUpStyles}>
                 <iframe
                   title="Guardian Email Sign-up Form"
-                  src={`https://profile.${domain}/email/form/footer/today-uk`}
+                  src={`https://www.theguardian.com/email/form/footer/today-uk`}
```

Comment:
just curious is the `www.` needed?

Reply:
That's a good question. On frontend the source is typically just `/email/form/footer/today-uk`, which I think then defaults to using the domain of the page. 

Is there any detriment to including `www.`? It's not something I've thought about much before

Reply:
Having just checked against PROD, not including the `www.` works the same way